tests/test_public_cli.py

```diff
@@ -247,6 +262,32 @@ def test_partial_run(
     ):
         public_cli.resume(database=tmp_db, run_id=1)
 
+    output = capsys.readouterr().out
+
+    # Now delete the runs, and confirm what is left behind...
+    public_cli.delete_run(database=tmp_db, run_id=1)
+    output = capsys.readouterr().out
+    assert "INFO: Run 1 contains 0/0=0² fastANI comparisons, status: Empty\n" in output
+
+    # Forcing as this has data
+    public_cli.delete_run(database=tmp_db, run_id=3, force=True)
+    output = capsys.readouterr().out
+    assert "INFO: Run 3 contains all 4=2² fastANI comparisons, status: Done\n" in output
+
+    # Finally delete run 2, this will assume last one remaining
+    public_cli.delete_run(database=tmp_db, force=True)
+    output = capsys.readouterr().out
+    assert (
+        "INFO: Deleting most recent run\n"
+        "INFO: Run 2 contains 4/9=3² fastANI comparisons, status: Running\n"
+    ) in output
+
+    assert session.query(db_orm.Run).count() == 0
+    assert session.query(db_orm.RunGenomeAssociation).count() == 0
+    assert session.query(db_orm.Configuration).count() == 1
+    assert session.query(db_orm.Genome).count() == 3  # noqa: PLR2004
+    assert session.query(db_orm.Comparison).count() == 4  # noqa: PLR2004
```

Comment:
In a future "aggressive" or "cascade" mode, all of these tables should become empty.